server/models/User.js

```diff
@@ -19,20 +21,21 @@ const userSchema = new mongoose.Schema({
   register_date: {
```

Comment:
Prefer camelCase in naming your variables

Reply:
ok